.travis.yml

```diff
@@ -50,4 +15,4 @@ script:
   - php docs/build-docs.php lint
 
 after_success:
-  - travis_retry php vendor/bin/coveralls -v
+  - travis_retry php vendor/bin/coveralls -v
```

Comment:
nit: revert?

Reply:
I don't remember removing that. I will take a look.